src/components/Nav/index.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+import * as S from './style';
+import * as I from '@/assets/imgs';
+import Image from 'next/image';
+import React, { useEffect, useState } from 'react';
+
+const Nav = () => {
+  const [domLoaded, setDomLoaded] = useState(false);
+
+  useEffect(() => {
+    setDomLoaded(true);
+  }, []);
+  return (
+    <>
+      {domLoaded && (
+        <S.NavContainer>
+          <S.IconContainer>
+            <Image src={I.LogoIcon} alt="" />
+            <S.Title>EveryGSM</S.Title>
+          </S.IconContainer>
+        </S.NavContainer>
+      )}
+    </>
+  );
```

Comment:
dom이 렌더링 되었을때 Nav를 보여지게 하는 이유가 있나요?

Reply:
서버에서 렌더링된 것과 UI가 일치하지 않는 오류를 해결하다가 시도한 방법인데 저 방법으로 땜빵 치려다가 이유를 찾고 정상적으로 해결하였습니다
layout.tsx에서 nav 태그를 body 태그와 html 태그 사이에 위치시켜서 생긴 제 실수였습니다

위 코드는 Del trash code 커밋에서 삭제 하였습니다
감사합니다!

[참고했던 문서입니다!](https://stackoverflow.com/questions/71706064/react-18-hydration-failed-because-the-initial-ui-does-not-match-what-was-render)